monticore-generator/src/main/java/de/monticore/codegen/cd2java/_parser/ParserClassDecorator.java

```diff
@@ -79,10 +86,22 @@ protected List<ASTCDMethod> createCreateMethods(String grammarName){
     this.replaceTemplate(EMPTY_BODY, createReader, new TemplateHookPoint(TEMPLATE_PATH + "CreateReader", grammarName));
     methods.add(createReader);
 
-
     return methods;
   }
 
+  protected List<ASTCDMember> createMode(){
+    List<ASTCDMember> members = Lists.newArrayList();
+
+    ASTCDAttribute modeAttribute = CDAttributeFacade.getInstance().createAttribute(PROTECTED.build(), "String", "mode");
```

Comment:
1. would this be more readable or incorrect?
2. add small doc to setter (ALu -> is this possible if using the MutatorDecorator?) to describe what it does (and where to get the mode-String-constant from?)
```suggestion
    ASTCDAttribute modeAttribute = CDAttributeFacade.getInstance().createAttribute(PROTECTED.build(), "String", "lexerMode");
```